server/models/Profile.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+const timeSlot = new mongoose.Schema({ start: Date, end: Date })
+
+const profileSchema = new mongoose.Schema({
+    firstname: { type: String },
+    lastname: { type: String },
+    description: { type: String },
+    availability: { type: [timeSlot] },
+    address: { type: String },
+    dateOfBirth: { type: Date },
+    photo: { type: String },
+    socialMedia: { type: String }
+});
```

Comment:
Could we add some more formatting here. 

```js
const profileSchema = new mongoose.Schema({
    firstname: { 
        type: String 
    },
    . . . 
});
```

Reply:
Got it, alongside additional sanitazion i will fix the formatting